### ISSUE TITLE
tsv-summarize: "total line" when grouping.

### ISSUE THREAD

**Issue by Wyatts** [eBay/tsv-utils-dlang#43]
Only a small ER this time. ;)

I hit a case where I'd like to have a "total" line in addition to the output of my --group-by.  I can get it by removing the --group-by option and rerunning the input, but it'd be nice if I could get them both in one go.

**Comment by jondegenhardt** [comment]
Makes sense. Not sure if it's a small ER or not :)  Depends on how I structured that part of the code. Hopefully small. I'll look into it.
